source/math/Quaternion.ooc

```diff
@@ -109,11 +109,11 @@ Quaternion: cover {
 	hamiltonProduct: static func (left, right: This) -> This {
 		(a1, b1, c1, d1) := (left w, left x, left y, left z)
 		(a2, b2, c2, d2) := (right w, right x, right y, right z)
-		w := a1 * a2 - b1 * b2 - c1 * c2 - d1 * d2;
-		x := a1 * b2 + b1 * a2 + c1 * d2 - d1 * c2;
-		y := a1 * c2 - b1 * d2 + c1 * a2 + d1 * b2;
-		z := a1 * d2 + b1 * c2 - c1 * b2 + d1 * a2;
-		return This new(w, x, y, z);
+		w := a1 * a2 - b1 * b2 - c1 * c2 - d1 * d2
+		x := a1 * b2 + b1 * a2 + c1 * d2 - d1 * c2
+		y := a1 * c2 - b1 * d2 + c1 * a2 + d1 * b2
+		z := a1 * d2 + b1 * c2 - c1 * b2 + d1 * a2
+		return This new(w, x, y, z)
```

Comment:
I vote no, we should take advantage of that fact that we don't have to use ```return``` in cases like this. It's quite clear without it.